Common/src/main/java/com/almostreliable/unified/config/DuplicateConfig.java

```diff
@@ -60,20 +60,35 @@ public boolean shouldIgnoreRecipe(RecipeLink recipe) {
      * @return True if the recipe type is ignored, false otherwise
      */
     private boolean isRecipeTypeIgnored(RecipeLink recipe) {
-        return ignoredRecipeTypesCache.computeIfAbsent(recipe.getType(), type -> {
```

Comment:
this gets called so much that the capturing lambda actually caused a minor performance hit, so I turned this `computeIfAbsent` into a regular `get/put` sequence